Sources/Cocoa/Extensions/UILabel+Extensions.swift

```diff
@@ -149,3 +149,32 @@ extension UILabel {
         )
     }
 }
+
+// MARK: Accessibility
+
+extension UILabel {
+    /// The accessibility trait value that the element takes based on the
+    /// text style it was given when created.
```

Comment:
```suggestion
    /// The accessibility trait value that the element takes based on the text style
    /// it was given when created.
```

Could you please update all comments in this pr to follow 80 column width?

Reply:
Done